gst/nnstreamer/meson.build

```diff
@@ -80,6 +80,13 @@ nnstreamer_shared = shared_library('nnstreamer',
   install_dir: plugins_install_dir
 )
 
+meson.add_install_script(
```

Comment:
how about using target name (other platforms)
```
# TODO target name using method name() (since 0.54.0)
# target_name = nnstreamer_shared.name()
libpath_split = nnstreamer_shared.full_path().split('/')
target_name = libpath_split[libpath_split.length() - 1]

meson.add_install_script(
  'sh', '-c',
  'ln -fs @0@ @1@@2@'.format(
    join_paths(plugins_install_dir, target_name),
    '${DESTDIR}',
    join_paths(nnstreamer_libdir, target_name)))
```

Reply:
Yup. That's better.

Reply:
And.. applied.